docs/modules/operation/pages/deep-dive/topology/enlinkd/layer-3-discovery.adoc

```diff
@@ -10,4 +10,8 @@ You can use the following routing daemons to provide links-based Layer 3 informa
 
 SNMP agents that support the appropriate MIBs provide information on Layer 2 network links.
 For this reason, you must have a working SNMP configuration.
-The link data that enlinkd discovers is detailed in the topology user interface, and on a node's details page.
+The following pages describe the MIBs required to allow link discovery:
```

Comment:
```suggestion

The following pages describe the MIBs required to allow link discovery:
```

Reply:
commit 32df5d3cb942ff101097397d8c88f31da932fcb1 (HEAD -> ar/jira/NMS-14909)
Author: Antonio <rssntn67@yahoo.it>
Date:   Tue Jan 10 10:23:16 2023 +0100

    Fixed space return